python/gps/gps_main.py

```diff
@@ -19,7 +19,7 @@
 from gps.utility.data_logger import DataLogger
 from gps.sample.sample_list import SampleList
 
-logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.INFO)
```

Comment:
Ok no problem. I've found DEBUG pretty useful, but I can just keep that as a local change.